sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -76,6 +77,13 @@ export class Notification extends RecordDataModel {
     nullable: true,
   })
   permanentFailureError: PermanentFailureError[];
+  /** Metadata information related to the saved notification. */
```

Comment:
I thought we should use the same comment style as in line 69 - 73. In addition to this, correct me if I'm wrong @andrewsignori-aot. I thought we need to use the same comment (with a small difference) on both the model here and the SQL file above.

Reply:
Sorry, missed it 😅